modules/mongodb/templates/mongodb-restore-s3.erb

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+S3_BUCKET=<%= @s3_bucket %>
+BACKUP_DIR=<%= @backup_dir %>
+PRIMARY=$(mongo --quiet --eval 'db.isMaster().ismaster')
+
+cd $BACKUP_DIR
+
+
+TIME="$(date +%s)"
+
+# Download and decrypt backup
+/usr/bin/envdir <%= @env_dir %>/env.d /usr/local/bin/s3cmd get --force `/usr/local/bin/s3cmd ls s3://${S3_BUCKET}/mongodump* | \
+tail -1 | awk '{print $4}'`| awk '{print $4}' | tr -d "'" | \
```

Comment:
Cool. I think Terraform has lifecycle rules for buckets so if you fancy adding [that to our terraform repo](https://github.com/alphagov/govuk-terraform-provisioning/) feel free ;)